.travis.yml

```diff
@@ -0,0 +1,20 @@
+os: linux
+dist: bionic
+arch: amd64
+
+jobs:
+  include:
+  - stage: Build
+    name: "Build dependencies"
+    services: docker
+    before_script:
+      - git clone https://github.com/binrats/gtirb
+      - git clone https://github.com/binrats/gtirb-pprinter
+      - docker pull binrats/ddisasm-base
+      - docker run -d -t --name souf -w /ddisasm --mount src="$(pwd)",target=/ddisasm,type=bind binrats/ddisasm-base /bin/sh
```

Comment:
Why the name `souf`?

Reply:
I changed the name to better reflect the purpose.